src/main/java/org/fhirvp/usecase/impl/PatientNotification.java

```diff
@@ -0,0 +1,41 @@
+package org.fhirvp.usecase.impl;
+
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Patient;
+import io.quarkus.logging.Log;
+import org.fhirvp.model.FHIRNotificationEvent;
+import org.fhirvp.ports.ActiveVaccinationSchemePort;
+import org.fhirvp.ports.PatientPort;
+import org.fhirvp.ports.impl.fhir.exception.FHIRServerException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class PatientNotification {
```

Comment:
Naming suggestion: `OnPatientNotificationUseCase`. The current name sounds like a model class to me.

Reply:
This UseCase should also have an interface definition to decouple the implementation from the classes that use this UseCase.

Reply:
Addressed at 86512c8